js/tests/builders.spec.ts

```diff
@@ -74,8 +74,8 @@ describe('messages builders', () => {
       body : quote
     })
 
-    expect(from).to.equal(rfqMessage.from)
-    expect(to).to.equal(rfqMessage.to)
+    expect(from).to.equal(rfqMessage.to)
```

Comment:
would this make more sense if we renamed to `sender` and `recipient`? then the logic would read like:

`expect(quoteSender).to.equal(rfqMessage.recipient)` which seems somewhat more english-y

Reply:
oh, i like that ^_^